features/dd-sdk-android-flags/src/test/kotlin/com/datadog/android/flags/internal/FlagsFeatureTest.kt

```diff
@@ -56,6 +60,7 @@ internal class FlagsFeatureTest {
     fun `set up`() {
         whenever(mockSdkCore.internalLogger) doReturn mockInternalLogger
         whenever(mockSdkCore.createSingleThreadExecutorService(any())) doReturn mockExecutorService
+//        whenever(mockSdkCore.getDatadogContext()) doReturn mockDatadogContext
```

Comment:
seems like this line should be removed and there is no actual usage of `mockDatadogContext` above?